README.md

```diff
@@ -167,3 +167,35 @@ Go to
 http://localhost:8000/api/users/tasks/  
 http://localhost:8000/api/users/tasks/1  
 
+## Points endpoints  
+There are currently several enpoints responsible for managing student points available:
+
+1. `api/users/points/` accesible with a `GET` request and followed by a query string pointing to a student primary key.  
+For example: `api/users/points/?studentId=2/` should return Point instances assigned to the student with `pk=2`  
```

Comment:
`api/users/points/?studentId=2/` this trailing slash is redundant and couses ValueError :)

Reply:
✅ done